src/config/express.js

```diff
@@ -57,6 +60,13 @@ if (config.staticPath) {
     app.use(express.static(config.staticPath));
 }
 
+if (config.filesPath) {
```

Comment:
Please remove this condition, server should always serve uploads.
front-end dist is another case in the reason that back-end can be started without front-end.